.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,51 +1,41 @@
 
 <!--
-^ Please summarise the changes you have done and explain why they are necessary here ^
-
-For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
-For new packages please briefly describe the package or provide a link to its homepage.
+Please summarise your changes and explain why they are necessary here.
 -->
 
 ## Things done
 
-<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->
+<!-- Please check what applies. Not all of these are hard requirements. -->
```

Comment:
I wonder if we can concisely convey the idea of "you can modify this checklist to better fit your PR, but the default checklist is designed to provide relevant information to reviewers, so make changes with care".

IIUC, no part of the template is compulsory, and some PRs will not really benefit from any of the checklist. We probably don't want to say that without _very_ careful wording, else we may encourage contributors to lazily ignore the checklist, even in the more typical scenario where it _is_ useful and relevant.